examples/scroll/README.md

```diff
@@ -0,0 +1,28 @@
+# Scroll Elements Into View
+
+**Test Code**: [index.js](index.js)  
+
+TestCafe [actions](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/interact-with-the-page.html) have a built-in scrolling mechanism. In rare cases you might need to scroll the page manually. This example demonstrates two common ways to do this: a `hover` action and a [ClientFunction](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html)
+
+The test page has 2 buttons. They are located 2000 px and 4000 px to the right respectively, both outside of viewport.
+
+The `index.js` file includes 2 tests.
+
+The first test scrolls the page with a [hover action](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/hover.html).
+
+The second test introduces 2 custom Client Functions that scroll the page:
+
+1. The `scrollToElement(selector, offsetX, offsetY)` function scrolls the page so that the element is inside the viewport.
```

Comment:
So that the element... you can't say that. Please don't skip words and steps.  The `scrollToElement(selector, offsetX, offsetY)` function scrolls the page to place the element is inside the viewport OR something else? Clarify.

Reply:
> The `scrollToElement(selector, offsetX, offsetY)` function scrolls the page **until** that element is inside the viewport.
OR
>The `scrollToElement(selector, offsetX, offsetY)` function scrolls the page to place the element inside the viewport.

Reply:
How can you scroll something until something appears somewhere? What am I missing here?

Reply:
> The `scrollToElement(selector, offsetX, offsetY)` function scrolls the element into the viewport.